docs/application/native/guides/multimedia/webrtc.md

```diff
@@ -0,0 +1,447 @@
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        unsigned int a_src_id;
+        unsigned int v_src_id;
+
+        ret = webrtc_create(&webrtc);
+        ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_AUDIOTEST, &a_src_id);
+        ret = webrtc_add_media_source(webrtc, WEBRTC_MEDIA_SOURCE_TYPE_VIDEOTEST, &v_src_id);
+        ...
+        ret = webrtc_set_ice_candidate_cb(webrtc, _ice_candidate_cb, user_data);
+        ret = webrtc_set_state_changed_cb(webrtc, _state_changed_cb, user_data);
+        ...
+        ret = webrtc_start(webrtc);
+    }
+    ```
+
+2. If the handle is an offerer, to create offer description, use `webrtc_create_offer()` or `webrtc_create_offer_async()` function:
+
+    ```c
+    int ret;
+    webrtc_h webrtc;
+    char *offer_desc;
+    ...
+    ret = webrtc_start(webrtc);
+    ret = webrtc_create_offer(webrtc, NULL, &offer_desc);
+    ...
+    /* After sending this offer description to the remote peer */
+    free(offer_desc);
+    ```
+
+3. If the handle is an answerer, to create answer description, use `webrtc_create_answer()` or `webrtc_create_answer_async()` function:
+
+    ```c
+    int ret;
+    webrtc_h webrtc;
+    char *answer_desc;
+    ...
+    ret = webrtc_start(webrtc);
+    ...
+    /* After receiving an offer description from the remote peer */
+    ret = webrtc_set_remote_description(webrtc, offer_desc_from_remote);
+    ret = webrtc_create_answer(webrtc, NULL, &answer_desc);
+    ...
+    /* After sending this answer description to the remote peer */
+    free(offer_desc);
+    ```
+
+4. To gather ICE candidates, use `webrtc_set_local_description()` function:
+
+    ```c
+    void _ice_candidate_cb(webrtc_h webrtc, const char *candidate, void *user_data)
+    {
+        /* Gather candidates or send candidate to the remote peer via signaling server */
+    }
+
+    void _ice_gathering_state_change_cb(webrtc_h webrtc, webrtc_ice_gathering_state_e state, void *user_data)
+    {
+        /* Until the state is changed to WEBRTC_ICE_GATHERING_STATE_COMPLETE, _ice_candidate_cb() will be called */
+    }
+
+    void webrtc_func(void)
+    {
+        int ret;
+        webrtc_h webrtc;
+        char *local_desc;
+        ...
+        ret = webrtc_set_ice_candidate_cb(webrtc, _ice_candidate_cb, user_data);
+        ret = webrtc_set_state_changed_cb(webrtc, _state_changed_cb, user_data);
+        ret = webrtc_set_ice_gathering_state_change_cb(webrtc, _ice_gathering_state_change_cb, user_data);
+        ...
+        ret = webrtc_start(webrtc);
+        ...
+        /* local_desc can be obtained from calling webrtc_create_offer() or webrtc_create_answer() */
+        ret = webrtc_set_local_description(webrtc, local_desc);
+    }
+    ```
+
+5. To finish the negotiation, use `webrtc_add_ice_candidate()`, `webrtc_set_local_description()` or `webrtc_set_remote_description()` function:
```

Comment:
Please avoid using the word "function" when function is already declared.
```suggestion
5. To finish the negotiation, use `webrtc_add_ice_candidate()`, `webrtc_set_local_description()` or `webrtc_set_remote_description()`:
```

Reply:
Okay, done.